.circleci/config.yml

```diff
@@ -135,7 +135,7 @@ jobs:
       - run:
           name: Test IDL generation
           working_directory: tests/idl_gen
-          command: cargo test --features mantle-build-test
+          command: ./run_tests.sh
```

Comment:
can this be a cargo command still eg `cargo test --target wasm32-wasi`.
Indirection to a shell script just makes it a bit harder for someone looking at how to replicate or run tests to figure out what the actual command is

Reply:
No because it's an integration test that requires a two-step build. One build to get the wasms and another build for the testing binary. I think that a shell script called `run_tests.sh` is fairy obvious--especially for something that isn't user facing.